Updating life section

Pat-Reen opened the pull request:
Adding in subsections on decision trees and bayesian applications.

JackyP commented:
Hi @Pat-Reen,

Thank you for the contribution.

Some minor issues picking up from review:
 * R: Life Industry Stats in # Tableau and R has an extra "#"
 * The new articles just say "Background" and "Overview"  on the sidebar, can you add a more descriptive title?
 * Libraries section in Bayes doesn't display the libraries
 * "Good interpretability and Robustness," has Robustness randomly capitalised
 * ``[Data description](Framingham Longitudinal Data Documentation.pdf)`` isn't showing up as a link. Is the pdf copyright available?
 * In the GBDT vs XGBoost section, I'd argue that XGBoost is a kind of GBDT, one that brings innovations to the algorithm. The [docs](https://xgboost.readthedocs.io/en/stable/) describe themselves that way too - "XGBoost provides a parallel tree boosting (also known as GBDT, GBM)". Perhaps it could be rephrased a little e.g. "Original GBDT vs XGBoost"?

Another general q's I had: How are you getting NN's to do well on sparse data vs GBDT?

I also just want to say it's really cool to see MCMC and R XGBoost on the cookbook.

Pat-Reen commented:
Replaced by pull request #36. Addresses the feedback from @JackyP.

Pat-Reen commented:
I have updated. Replaced with PR #36.

On Tue, 15 Mar 2022, 11:41 am Patrick Reen, ***@***.***> wrote:

> Thanks both. I owe you some edits as well. Will get to in soon. Thanks!
>
> On Tue, 15 Mar 2022, 9:03 am Charlie, ***@***.***> wrote:
>
>> Thanks Jackie for review. I have made some changes to the Tree model
>> section and will rephrase the description. Agree that xgboost is an
>> improved algorithm of GBDT.
>>
>> I haven't spent too much time on training life data using NN and the
>> preliminary result was worse than tree models. I would expect NN requires
>> dense data and large dataset to work well. I will speak to Pat about this
>> and see if we can add something.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/ActuariesInstitute/cookbook/pull/32#issuecomment-1067346731>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AV2VKIH4XGGCC2QYPOZXNCDU76ZUTANCNFSM5NQVDHKA>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

Pat-Reen commented:
Thanks both. I owe you some edits as well. Will get to in soon. Thanks!

On Tue, 15 Mar 2022, 9:03 am Charlie, ***@***.***> wrote:

> Thanks Jackie for review. I have made some changes to the Tree model
> section and will rephrase the description. Agree that xgboost is an
> improved algorithm of GBDT.
>
> I haven't spent too much time on training life data using NN and the
> preliminary result was worse than tree models. I would expect NN requires
> dense data and large dataset to work well. I will speak to Pat about this
> and see if we can add something.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ActuariesInstitute/cookbook/pull/32#issuecomment-1067346731>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AV2VKIH4XGGCC2QYPOZXNCDU76ZUTANCNFSM5NQVDHKA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
